app/models/event.rb

```diff
@@ -1,58 +0,0 @@
-class Event < ApplicationRecord
```

Comment:
I agree that it's not wise to remove the underlying table now, but would be good to capture this action in a Jira ticket, Github issue or a code TODO somehow so it doesn't get forgotten forever.

Reply:
Re the TODO I was thinking I'd rename the GenericEvent model -> Event and inidicate in the table name. Since we're not doing that it makes sense to capture it in a ticket I think since there isn't a logical place to say it